message-store/message-store.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina/http;
+
+type FirebaseResponse record {|
+    string address?;
+    record {}[] results?;
```

Comment:
Is it possible to replace the type of `result` to `json`

Reply:
yes.